src/plugins/AutoRefresh.ts

```diff
@@ -47,7 +49,7 @@ class AutoRefresh extends GenericPlugin {
       return;
     }
 
-    const { refreshRateInSeconds = 30 } = this.ad.configuration;
+    const { refreshRateInSeconds = 30, refreshLimit = null } = this.ad.configuration;
```

Comment:
Why set. this to null here?

If it is not defined it will be falsey already.

There could be a problem if 0 was ever passed as a valid amount but in that case there would not be an ad, right?

Reply:
I was following what the docs say. How should we proceed? 

<img width="986" alt="Screen Shot 2020-12-02 at 10 50 13 AM" src="https://user-images.githubusercontent.com/5769616/100917576-2b36a780-348c-11eb-82e6-0a2ce63e9ae5.png">

Reply:
I guess null is fine. DOnt think it is of much consequence